metastore/src/java/org/apache/hadoop/hive/metastore/MetaStoreUtils.java

```diff
@@ -1269,7 +1269,7 @@ public static void startMetaStore(final int port,
       @Override
       public void run() {
         try {
-          HiveMetaStore.startMetaStore(port, bridge, finalHiveConf);
+          HiveMetaStore.startMetaStore(null, port, bridge, finalHiveConf);
```

Comment:
Seems to be mostly used by tests so I opted to changing this to `null` so that the existing behavior of binding to `0.0.0.0` is retained for code using the MetastoreUtils class to start a metastore for tests.